profile.js

```diff
@@ -35,4 +35,113 @@ document.getElementById('menu').addEventListener('click', () => {
   document.getElementById('link1').addEventListener('click', hideMobMenu);
   document.getElementById('link2').addEventListener('click', hideMobMenu);
   document.getElementById('link3').addEventListener('click', hideMobMenu);
-});
\ No newline at end of file
+});
+// || =========== Pop-Up Modal ============ ||*/
+const arrProjects = [];
```

Comment:
- [ ] [optional] Kindly check the design of your project at different widths it changes breaking the design.